tests/test_dagbuilder.py

```diff
@@ -461,7 +454,7 @@ def test_build():
         assert actual["dag"].tags == ["tag1", "tag2"]
 
 
-def test_get_dag_params():
+def test_get_dag_params_default_args():
```

Comment:
```suggestion
def test_get_dag_params_dag_with_task_group():
```

I am not sure `default_args` if a good suffix here. The other test `test_get_dag_params` also gets `default_args`. The difference in both the tests is that the other one passes the `DAG_CONFIG` and this one passes `DAG_CONFIG_TASK_GROUP` to DagBuilder.